docs/user-guide/connecting.md

```diff
@@ -222,9 +222,42 @@ Current MacOS systems do not have an X window system. Users should install the X
 
 ## Host Keys
 
-A host key adds an extra security layer for users over SSH. Using one enables users to log in to ARCHER2 if the login node has the same corresponding host key. The host key can be found in the entries in `~/.ssh/known_hosts`, as given in the example below: 
+A host key adds an extra security layer for users over SSH. Using one enables users to log in to ARCHER2 if the login node has the same corresponding host key. 
```

Comment:
I am not sure I am quite getting what this is getting at here. Possibly something like this would be more useful to users:

> Adding the host keys to your SSH configuration file provides an extra level of security for your connections to ARCHER2. The host keys are checked against the login nodes when you login to ARCHER2 and if the remote server key does not match the one in the configuration file, the connection will be refused. This provides protection against potential malicious servers masquerading as the ARCHER2 login nodes.